opendrift/models/basemodel/init.py

```diff
@@ -21,9 +21,46 @@ class Init(State, Configurable):
 
     env: Environment
 
-    def __init__(self):
+    def __init__(self,
+                 loglevel=logging.DEBUG, logtime='%H:%M:%S', logfile=None):
         super().__init__()
 
+        if loglevel != 'custom':
```

Comment:
I think we can keep this now, but I would prefer to keep logging setup not part of the Simulation setup code. Maybe one solution is to have `opendrift.install_logs(...)` as a new module method. Otherwise things will be a bit messy when you create multiple simulation objects and initializing a Simulation has side-effects not really related to the exact simulation. Maybe we could make a compromise where this calls into `opendrift.install_logs` if a loglevel is supplied.